front/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "build": "react-scripts build",
-    "format": "prettier --write \"**/*.{js,scss,json,ts,tsx}\"",
+    "format": "eslint --fix --ext .tsx,.js . && prettier --write \"**/*.{js,scss,json,ts,tsx}\"",
```

Comment:
Great stuff :) can you add `.ts` to the eslint extensions?

Reply:
done https://github.com/trybick/tv-minder/pull/50/commits/c2d4ae55c09bf70f7d8f513c6f32851b02c46f8d